# 加成区信息

# 是否激活
bool isActive

# Buff区类型
# 未知
uint8 TYPE_NONE = 0
# 红方回血区
uint8 TYPE_RED_RESTORATION = 1
# 红方弹药补给区
uint8 TYPE_RED_SUPPLY = 2
# 蓝方回血区
uint8 TYPE_BLUE_RESTORATION = 3
# 蓝方弹药补给区
uint8 TYPE_BLUE_SUPPLY = 4
# 禁止射击区
uint8 TYPE_NO_SHOOTING = 5
# 禁止移动区
uint8 TYPE_NO_MOVING = 6
uint8 type